modelling/src/main/java/org/axonframework/modelling/saga/SagaMethodMessageHandlingMember.java

```diff
@@ -91,10 +90,12 @@ public SagaCreationPolicy getCreationPolicy() {
     /**
      * Indicates whether this handler is one that ends the Saga lifecycle
      *
-     * @return {@code true} if the Saga lifecycle ends unconditionally after this call, otherwise
-     * {@code false}
+     * @return {@code true} if the Saga lifecycle ends unconditionally after this call, otherwise {@code false}
+     * @deprecated in favor of the dedicated {@link EndSagaMessageHandlerDefinition} and {@link
+     * EndSagaMessageHandlerDefinition.EndSageMessageHandlingMember}
      */
+    @Deprecated
```

Comment:
Deprecating this doesn't make sense. The method is set to return false in any case. That breaks backward compatibility already. Rather break it through a compilation error, than through a subtle change at runtime.

Since this is not a public API method, I suggest removing the it.

Reply:
Exactly why I had the constructor still present in the previous review. But sure, I'll also remove this method.